overlord/snapstate/export_test.go

```diff
@@ -149,20 +149,20 @@ func ByKindOrder(snaps ...*snap.Info) []*snap.Info {
 }
 
 func MockModelWithBase(baseName string) (restore func()) {
-	return mockModel(baseName)
+	return mockModel(fmt.Sprintf("\nbase: %s", baseName))
```

Comment:
The leading newline looks odd. What is its purpose? Maybe that could be a comment in case other people might wonder.